googliser-completion

```diff
@@ -9,7 +9,7 @@ _GoogliserCompletion()
   --exact-search --help --lightning --links-only --no-colour --no-color --safesearch-off \
   --quiet --random --reindex-rename --save-links --skip-no-size --aspect-ratio \
   --border-pixels --colour --color --exclude-links --exclude-words --format --gallery \
-  --input-links --input-phrases --lower-size --minimum-pixels --number --output --parallel \
+  --input-links --input-phrases --install --lower-size --minimum-pixels --number --output --parallel \
```

Comment:
Will `--install` be needed here? User won't have access to this as a tab-completion option until install has been done. Afterwards, there's no-need to install again. ;)

Reply:
True  True